core/src/test/java/gov/nist/secauto/metaschema/core/model/xml/ModuleLoaderTest.java

```diff
@@ -72,7 +72,7 @@ void testUrl() throws MetaschemaException, IOException { // NOPMD - intentional
   void testFile() throws MetaschemaException, IOException {
     ModuleLoader loader = new ModuleLoader();
     URI moduleUri = ObjectUtils.notNull(
-        Paths.get("metaschema/test-suite/docs-models/models_metaschema.xml").toUri());
+        Paths.get("src/test/resources/content/custom-entity-metaschema.xml").toUri());
```

Comment:
I seem to be unable to find this file in the branch, am I missing something? I wanted to see the Metaschema module to understand the tests.

Reply:
It already [exists](https://github.com/david-waltermire-nist/metaschema-java/blob/feature-support-numeric-indexes/core/src/test/resources/content/custom-entity-metaschema.xml) in the branch.

Reply:
I had trouble finding it for naively yesterday using the GitHub interface and I was in between tasks so I hadn't pulled down the repo, I apologize. I am off to more thorough review this afternoon.